velox/common/caching/SimpleLRUCache.h

```diff
@@ -18,271 +18,145 @@
 #include <cstdint>
 #include <functional>
 #include <list>
+#include <optional>
 
-#include "folly/container/F14Map.h"
+#include "folly/container/EvictingCacheMap.h"
 #include "glog/logging.h"
 
 namespace facebook::velox {
 
 struct SimpleLRUCacheStats {
+  SimpleLRUCacheStats(
+      size_t maxSize,
```

Comment:
```
s/maxSize/_maxSize/

dittos
```

Reply:
do we append underscore in front of stats entries now?

Reply:
We do that if the input have the same names as the structure member? It is used in Velox now.